### ISSUE TITLE
any way to upgrade GPU-driver/browser acceleration?

### ISSUE THREAD

**Issue by PhinPhin** [Screenly/screenly-ose#644]
Hi,

I tried to activate the new GL-driver in raspi-config, though this gives only black screens with audio instead of videoplayback.

I have a canvas-animation that is lagging in the browser, which is critical for the project i use Screenly for, though. Any way to activate 2D-acceleration in the browser? In Chromium with activated 2D-canvas acceleration it works fine on the Pi3. I feel like UZBL is getting outdated for this kind of software....

Beside that: did read there will be a new Screenly-version, I doubt this will mean anything new for OSE? It's really a shame that Screenly OSE seems to be the only offline-digitalsignage solution out there. :/

**Comment by vpetersson** [comment]
Duplicate of #527.

tl;dr - There is no good replacement for UZBL. None of the other major browsers support the same degree of control that it does. That said, we do keep our eyes open and contributions are welcome.
